File: 백준/Bronze/2480. 주사위 세개/주사위 세개.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.on("line", (v) => {
  arr = v.split(" ");
  if (arr[0] === arr[1] && arr[1] === arr[2]) {
    console.log(10000 + arr[0] * 1000);
  } else if (arr[0] === arr[1] && arr[0] !== arr[2]) {
    console.log(1000 + arr[0] * 100);
  } else if (arr[0] !== arr[1] && arr[0] === arr[2]) {
    console.log(1000 + arr[0] * 100);
  } else if (arr[0] !== arr[1] && arr[1] === arr[2]) {
    console.log(1000 + arr[1] * 100);
  } else {
    console.log(Math.max(...arr.map((v) => +v)) * 100);
  }

  rl.close();
});

rl.on("close", () => {
  process.exit();
});

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
A, B, C = input().split()
A = int(A)
B = int(B)
C = int(C)

# 같은 눈이 3개가 나오면 10,000원+(같은 눈)×1,000
if(A==B and A==C):
    print(10000+(A*1000))
# 모두 다른 눈이 나오는 경우에는 (그 중 가장 큰 눈)×100
elif(A!=B and A!=C and B!=C):
    arr = [A,B,C]
    arr.sort(reverse=True)
    print(arr[0]*100)
# 같은 눈이 2개만 나오는 경우에는 1,000원+(같은 눈)×100
else:
    if(A==B):
        print(1000+(A*100))
    elif(A==C):
        print(1000+(A*100))
    else:
        print(1000+(B*100))